test/System.CommandLine.StaticCompletions.Tests/VerifyConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.CommandLine.StaticCompletions.Tests;
+
+using System.Runtime.CompilerServices;
+
+public static class VerifyConfiguration
+{
+    [ModuleInitializer]
+    public static void Initialize()
+    {
+        VerifyDiffPlex.Initialize(VerifyTests.DiffPlex.OutputType.Compact);
+
+        if (IsCI())
```

Comment:
just an FYI. u could use BuildServerDetector.Detected here https://github.com/VerifyTests/DiffEngine?tab=readme-ov-file#buildserverdetector

Reply:
Excellent pointer - added it to my tests' code.